SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/DormancyTest.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "DormancyTest.h"
+
+#include "DormancyTestActor.h"
+
+AActor* ADormancyTest::CreateDormancyTestActor()
+{
+	return GetWorld()->SpawnActor<ADormancyTestActor>({ 0.0f, 0.0f, 0.0f }, FRotator::ZeroRotator);
+}
+
+void ADormancyTest::RequireDormancyAndRepProperty(const TEnumAsByte<enum ENetDormancy> ExpectedNetDormancy, const int ExpectedTestIntProp)
+{
+	for (TActorIterator<ADormancyTestActor> Iter(GetWorld()); Iter; ++Iter)
+	{
+		const ADormancyTestActor* DormancyTestActor = *Iter;
+		RequireEqual_Int(DormancyTestActor->NetDormancy, ExpectedNetDormancy, TEXT("Dormancy on ADormancyTestActor"));
```

Comment:
oh no, I just realized something.... requires are mapped based on the string, so if you ever test for more than two dormancy actors, they will overwrite each other.........
We should fix that in the test framework at some point.

Reply:
Want me to append the actors name to the string or should we just wait for the internal fix?

Reply:
The proper fix is not planned at the moment, but that being said, this will not actually pose problems for now.
You can add the actor name as a possible improvement.